src/Acheve.TestHost/Routing/Tokenizers/PrimitiveParameterActionTokenizer.cs

```diff
@@ -32,10 +32,31 @@ public void AddTokens<TController>(TestServerAction action, TestServerTokenColle
                         tokens.AddToken(tokenName, tokenValue.ToString(), isConventional: false);
                     }
                 }
+                else if (parameters[i].ParameterType.IsArray
+                   && !IgnoreHeader(parameters[i]))
+                {
+                    dynamic tokenValue = action.ArgumentValues[i].Instance;
+
+                    if (tokenValue != null 
+                        && tokenValue.Length != 0 
+                        && (tokenValue[0].GetType().IsPrimitive
```

Comment:
Maybe you can use "parameters[i].ParameterType.GetElementType"

Reply:
i can't get the type inside an array :(

![imagen](https://user-images.githubusercontent.com/9169130/159033490-c694fbcc-6054-45a2-96d7-6615da48030f.png)

Reply:
`		
int[] array = new int[] { 1, 2, 3};
		
Console.WriteLine(array.GetType()); // System.Int32[]
Console.WriteLine(array.GetType().GetElementType()); // System.Int32
Console.WriteLine(array.GetType().GetElementType() == typeof(int)); // True
`
This code works

Reply:
You can add the type check in a function to use it on line 18 as well

Reply:
Done!
Thx for info and this page :)